README.md

```diff
@@ -318,6 +318,20 @@ interface State<T> {
 
 The downlevel .d.ts omits the variance annotations, which will change the variance in the cases where they were added because the compiler gets it wrong.
 
+### `typeof A<T>` (4.7)
+
+Typescript 4.7 supports type arguments for `typeof`:
+
+```ts
+type N = typeof func<number>;
+```
+
+becomes:
+
+```ts
+type N = any;
+```
```

Comment:
I'm struggling to write a good semantics part, not sure about the chosen approach

Reply:
The current architecture of downlevel-dts is syntax-only so it doesn't have the information to make this work. If the tool had type information, it could use an inlining strategy: write out the signature type after instantiation.

Constructing a full program to collection type information would make downlevel-dts much slower and more fragile.  So I think it's better not to downlevel this construction.

Reply:
... and to avoid usage of `typeof foo<T>` in the original code?